tests/conftest.py

```
@@ -4,11 +4,18 @@
 """
 import asyncio
 import pytest
-from httpx import AsyncClient
+from httpx import AsyncClient, ASGITransport
+from asgi_lifespan import LifespanManager
+from sqlalchemy.ext.asyncio import create_async_engine
+from app.core.config import get_settings
+from sqlalchemy import text
```

Ruff (0.11.9)

`sqlalchemy.text` imported but unused

Remove unused import: `sqlalchemy.text`

(F401)